jcapi.go

```diff
@@ -240,26 +273,31 @@ func (jc JCAPI) extractStringArray(input []interface{}) []string {
 	return returnVal
 }
```

Comment:
If you're touching this code anyway, like `getStringOrNil` below, you can declare `returnVal` in the method signatures return arguments, and avoid the var line here.